tests/unit/test_business_logic.py

```diff
@@ -1539,14 +1583,49 @@ def test_reset_review_file_no_reviews(bll):
         release_request=release_request, to_status=RequestStatus.SUBMITTED, user=author
     )
 
-    current_reviews = _get_current_file_reviews(bll, release_request, path, checker)
-    assert len(current_reviews) == 0
+    rfile = _get_request_file(bll, release_request, path)
+    assert rfile.status_for_user(checker) is None
+    assert rfile.status() == RequestFileReviewStatus.INCOMPLETE
 
     with pytest.raises(bll.FileReviewNotFound):
         bll.reset_review_file(release_request, path, checker)
 
-    current_reviews = _get_current_file_reviews(bll, release_request, path, checker)
-    assert len(current_reviews) == 0
+    rfile = _get_request_file(bll, release_request, path)
+    assert rfile.status_for_user(checker) is None
+    assert rfile.status() == RequestFileReviewStatus.INCOMPLETE
+
+
+@pytest.mark.parametrize(
+    "reviews, final_review",
+    [
+        (["APPROVED", "APPROVED"], "APPROVED"),
+        (["REJECTED", "REJECTED"], "REJECTED"),
+        (["APPROVED", "REJECTED"], "CONFLICTED"),
```

Comment:
Maybe also check the UNDECIDED status that a previously rejected file review could be in after a returned request is resubmitted?

Reply:
ooh, good shout

Reply:
Yeah, because UNDECIDED didn't exist when I started this, I'd missed handling it.

I've added a separate test, and UNDECIDED votes count as INCOMPLETE